src/portal/src/css/dark-theme.scss

```diff
@@ -0,0 +1,113 @@
+@import "../../node_modules/@clr/ui/clr-ui-dark.min.css";
+$dark-background-color: rgb(27, 42, 50) !important;
+$dark-font-color1: #acbac3 !important;
+$dark-font-color-title1: #eaedf0 !important;
+
+.label-form {
+  background-color: #212129 !important;
+}
+.esxc-gauge-circle-bgc-dark {
+  background-color: $dark-background-color;
+}
+
+.harbor-icon {
+  filter: drop-shadow(#ccc 58px 2px) !important;
+}
+
+.color-white-dark {
+  color: white;
+}
+
+.labelFilterPanel {
```

Comment:
Why here using cameral case? not dash?

Reply:
This is a historical reason, I have modified